### ISSUE TITLE
Class dompdf.wrapper does not exist

### ISSUE THREAD

**Issue by astroanu** [barryvdh/laravel-dompdf#130]
After the last composer update i get this.. 

ReflectionException in Container.php line 779:
Class dompdf.wrapper does not exist

in Container.php line 779
at ReflectionClass->__construct('dompdf.wrapper') in Container.php line 779
at Container->build('dompdf.wrapper', array()) in Container.php line 659
at Container->make('dompdf.wrapper', array()) in Application.php line 644
at Application->make('dompdf.wrapper') in Facade.php line 20

**Comment by barryvdh** [comment]
What version are you one? Can you try either 0.5.2 or 0.6.0?

**Comment by astroanu** [comment]
0.5.2 reverted to 0.5.0 and it worked yesterday. I'll try 0.6.0

**Comment by shinichi2510** [comment]
My composer.json
"php": ">=5.5.9",
        "laravel/framework": "5.1.*",
        "barryvdh/laravel-dompdf": "0.6.0"

in config/app.php
Barryvdh\DomPDF\ServiceProvider::class
'PDF'       => Barryvdh\DomPDF\Facade::class

I get the error
ReflectionException in Container.php line 737: Class dompdf.wrapper does not exist

    in Container.php line 737
    at ReflectionClass->__construct('dompdf.wrapper') in Container.php line 737
    at Container->build('dompdf.wrapper', array()) in Container.php line 627
    at Container->make('dompdf.wrapper', array()) in Application.php line 674
    at Application->make('dompdf.wrapper') in Facade.php line 20

Please help me

**Comment by amandatmt** [comment]
shinichi2510, did you get a solution?

**Comment by GregKeaney** [comment]
Anyone solve this??

**Comment by astroanu** [comment]
Do a `composer dump-autoload -o` it should work.

**Comment by GregKeaney** [comment]
Thanks but no luck still getting this;

ReflectionException in Container.php line 741:
Class dompdf.wrapper does not exist

**Comment by astroanu** [comment]
@GregKeaney hey, are you using the service provider?

**Comment by GregKeaney** [comment]
I am yes.  Pretty odd.  I actually started from scratch with a new install of Laravel 5.2 to see if I could get it to work.  It did.  So obviously something is gummed up with my actual project.  I am going to revert to  a prior state of that project and try the install again.  Stay tuned...

Thanks.

**Comment by GregKeaney** [comment]
This is troubling.  I have removed the package fully and reinstalled into my 5.2 project.  Issue is still there.  If I inject into controller like so;

use Barryvdh\DomPDF\Facade as PDF;

I get this error;

ReflectionException in Container.php line 741:
Class dompdf.wrapper does not exist

This is not the case for my clean install of 5.2.  How can I trouble shoot the issue/difference between the to installs?

**Comment by GregKeaney** [comment]
Holy s**t snacks.

So, I ran this;

php artisan cache:clear
and
php artisan config:cache

Then I got this;

ErrorException in AdobeFontMetrics.php line 45:
fopen(C:\xampp\htdocs\project\storage\fonts/\88396d120b8df37f1c66249eb40dc0fa.ufm): failed to open stream: No such file or directory

Then I added a writeable font dir to storage and got this;

Exception in Inline.php line 48:
No block-level parent found. Not good.

I then removed the CSS from the view I was trying to convert to PDF et voila - it worked!

Good news but - seriously - that seemed convoluted.

Thanks to all - all hail the community.
